app/src/main/java/com/example/beni/myapplication/RepositoriesActivity.java

```diff
@@ -0,0 +1,129 @@
+        mRecyclerView = (RecyclerView) findViewById(R.id.recycler_view);
+        mRecyclerView.setLayoutManager(new LinearLayoutManager(this));
+
+        Call<List<Repository>> repoCall = GitHub.Service.get().getRepos(preferences.getString("auth",null));
+        repoCall.enqueue(new Callback<List<Repository>>() {
+            @Override
+            public void onResponse(Call<List<Repository>> call, Response<List<Repository>> response) {
+                if(response.isSuccessful()){
+
+                    List<Repository> repos = response.body();
+                    Adapter adapter = new Adapter();
+                    adapter.setmData(repos);
+                    mRecyclerView.setAdapter(adapter);
+                }
+            }
+
+            @Override
+            public void onFailure(Call<List<Repository>> call, Throwable t) {
+
+            }
+        });
+
+
+
+    }
+
+    static class Adapter extends RecyclerView.Adapter{
+        List<Repository> mData;
+
+        public void setmData(List<Repository> mData) {
+            this.mData = mData;
+        }
+
+        @Override
+        public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+            View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.view_repository,parent,false);
+            return new ViewHolder(view);
+            }
+
+        @Override
+        public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
+            ((ViewHolder)holder).bind(mData.get(position));
+        }
+
+        @Override
+        public int getItemCount() {
+            return mData != null ? mData.size() : 0;
+        }
+
+        static class ViewHolder extends RecyclerView.ViewHolder {
+
+            private final TextView mNameAndOwner;
+            private final TextView mCountWatchers;
+            private final TextView mDescription;
+            private final CheckBox mIsPublic;
+            private final LinearLayout mTopics;
+
+            public ViewHolder(View itemView) {
+                super(itemView);
+                mCountWatchers = (TextView)itemView.findViewById(R.id.watchers);
+                mNameAndOwner = (TextView)itemView.findViewById(R.id.project_name);
+                mDescription = (TextView)itemView.findViewById(R.id.project_description);
+                mIsPublic = (CheckBox)itemView.findViewById(R.id.is_public);
+                mTopics = (LinearLayout)itemView.findViewById(R.id.expand_layout);
+
+            }
+            public void bind(Repository repository){
+                if(repository.getWatchersCount()>0)
+                    mCountWatchers.setText(String.valueOf(repository.getWatchersCount()));
+                else{
+                    mCountWatchers.setText("0");
+                }
+                mNameAndOwner.setText(itemView.getContext().getString(R.string.repo_name_owner,repository.getName(),repository.getOwner()));
```

Comment:
repository.getOwner() is an object not a String. That's why in the UI you'll see the default Object's to String for owner. I think you want repository.getOwner().getName().